src/Tasks/Microsoft.NET.Build.Tasks/PreprocessPackageDependenciesDesignTime.cs

```diff
@@ -52,6 +54,9 @@ public class PreprocessPackageDependenciesDesignTime : TaskBase
         private Dictionary<string, ItemMetadata> Assemblies { get; set; }
                     = new Dictionary<string, ItemMetadata>(StringComparer.OrdinalIgnoreCase);
 
+        private Dictionary<string, ItemMetadata> DiagnosticsMap { get; set; }
+                    = new Dictionary<string, ItemMetadata>(StringComparer.OrdinalIgnoreCase);
+
```

Comment:
Pre-existing, but why do we have private properties instead of private fields with `_name`?